Pcap++/header/PcapRemoteDevice.h

```diff
@@ -86,18 +86,18 @@ namespace pcpp
 		// c'tor is private, as only PcapRemoteDeviceList should create instances of it, and it'll create only one for every remote interface
 		PcapRemoteDevice(pcap_if_t* iface, PcapRemoteAuthentication* remoteAuthentication, const IPAddress& remoteMachineIP, uint16_t remoteMachinePort);
 
-		// private copy c'tor
-		PcapRemoteDevice( const PcapRemoteDevice& other );
-		// private assignment operator
-		PcapRemoteDevice& operator=(const PcapRemoteDevice& other);
-
 		static void* remoteDeviceCaptureThreadMain(void *ptr);
 
 		//overridden methods
 		ThreadStart getCaptureThreadStart();
 
 	public:
-		virtual ~PcapRemoteDevice() {}
+		PcapRemoteDevice(const PcapRemoteDevice&) = delete;
+		PcapRemoteDevice(PcapRemoteDevice&&) noexcept = delete;
```

Comment:
is `noexcept` necessary here?

Reply:
Strictly speaking, no. But most move ctors are marked `noexcept`, so I prefer to have the full signature.

Reply:
I see, sure.